packages/consumer/__snapshots__/build.test.ts.snap

```diff
@@ -0,0 +1,108 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`build side-effects 1`] = `
```

Comment:
Having these as reference will be of great help in Crackle https://github.com/seek-oss/crackle/blob/6c0f1bc76167a3a8c418bffc30e82879395b4ce7/packages/core/src/package-utils/bundle.ts#L46-L59